### PR TITLE
tests: enable functional integration test

### PR DESCRIPTION
**Pull request by kmr-srbh**

### CODE REVIEW

**Comment by kmr-srbh** [comment]
The `intrinsics_211.f90` test fails only on macOS.

File: integration_tests/CMakeLists.txt [hunk 1]
```diff
@@ -802,7 +802,7 @@ RUN(NAME intrinsics_207 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc) # double
 RUN(NAME intrinsics_208 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc fortran) # bit_size
 RUN(NAME intrinsics_209 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc) # atand
 RUN(NAME intrinsics_210 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc EXTRAFILES lcompilers_test_module.f90) # bessel_jn
-# RUN(NAME intrinsics_211 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc) # bessel_yn
+RUN(NAME intrinsics_211 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc) # bessel_yn
```
Comment (Pranavchiku) [inline integration_tests/CMakeLists.txt:805]: This is problem in `main` as well. I don't think we need to worry about it

Comment (Pranavchiku) [reply integration_tests/CMakeLists.txt:805]: can you confirm this @kmr-srbh?

Comment (kmr-srbh) [reply integration_tests/CMakeLists.txt:805]: You are right @Pranavchiku. I will revert this.

**Pull request merged**